rpcs3/Emu/RSX/VK/VKVertexBuffers.cpp

```diff
@@ -32,6 +32,7 @@ namespace vk
 		case rsx::primitive_type::polygon:
 			requires_modification = true;
 			return VK_PRIMITIVE_TOPOLOGY_TRIANGLE_LIST;
+		case rsx::primitive_type::invalid:
```

Comment:
I mean, what's the point in having default then?

Reply:
It's just some random warning and I wasn't in the mood to discriminate.

Anyway. What about the gl and vk switches?
I only added those changes because they seemed suspicious